### DISCUSSION TITLE
Bambu Studio 404 Error

### DISCUSSION THREAD

**Discussion by jeepwalker** [community/community#125272] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Get a 404 error when I try to access this page to download Bambu Studio:

https://github.com/bambulab/BambuStudio/releases/download/v01.09.01.66/Bambu_Studio_win_public-v01.09.01.66-20240511203248.exe

Link from https://bambulab.com/en/download

**Comment by Mattias3476** [comment]
Any solution? Me same problem.😭

**Comment by shrawank039** [comment]
most probably that version is not there anymore.
use this - https://github.com/bambulab/BambuStudio/releases/download/v01.09.01.66/Bambu_Studio_win_public-v01.09.01.67-20240516173854.exe
